modules/installation-special-config-storage.adoc

```diff
@@ -54,21 +54,22 @@ metadata:
   labels:
     machineconfiguration.openshift.io/role: worker
 boot_device:
-  layout: x86_64
+  layout: x86_64 <1>
   luks:
-    tpm2: true <1>
-    tang: <2>
+    tpm2: true <2>
+    tang: <3>
       - url: http://tang1.example.com:7500
         thumbprint: jwGN5tRFK-kF6pIX89ssF3khxxX
       - url: http://tang2.example.com:7500
         thumbprint: VCJsvZFjBSIHSldw78rOrq7h2ZF
-    threshold: 2 <3>
+    threshold: 2 <4>
 openshift:
   fips: true
 ----
-<1> Include this field if you want to use a Trusted Platform Module (TPM) to encrypt the root file system.
-<2> Include this section if you want to use one or more Tang servers.
-<3> Specify the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur.
+<1> Set this field to the instruction set architecture of the cluster nodes. Some examples include, `x86_64`, `aarch64`, `ppc64le` and others. 
```

Comment:
Thank you for the feedback @aleskandro! I know what you mean with the example yamls, but I think that this new sentence is more general and can still work with the example `x86_64` in the field. I'm leaning towards this update as well bc we usually want the formating of the sample yaml/ fields to be consistent. What do you think?

Reply:
Currently re wording that last sentence.